### PR TITLE
Warn when unauthorized email set as from email [MAILPOET-4253]

### PR DESCRIPTION
**Pull request by websupporter**
Fixes [MAILPOET-4253]

How To Test:
* Activate MSS sending method
* Change your from address in the Basic tab
* When your email is not authorized a message should appear once you leave the from address input field and you shouldn't be able to save
* When your email is authorized no message should appear once you leave the from address input field and you should be able to save
image

[MAILPOET-4253]: https://mailpoet.atlassian.net/browse/MAILPOET-4253?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/lib/Services/Bridge/API.php [hunk 1]
```diff
@@ -6,7 +6,8 @@
 use MailPoet\WP\Functions as WPFunctions;
 use WP_Error;
 
-class API {
+class
```
Comment (lysyjan) [inline mailpoet/lib/Services/Bridge/API.php]: This new line before the name of the class is probably a typo. Right?

File: mailpoet/tests/acceptance/Settings/AuthorizedEmailAddressesValidationCest.php [hunk 2]
```diff
@@ -8,50 +8,57 @@
...
     $i->wantTo('Error message disappears after email is replaced with authorized email');
+    $i->clearField('[data-automation-id="from-email-field"]');
     $i->fillField('[data-automation-id="from-email-field"]', \AcceptanceTester::AUTHORIZED_SENDING_EMAIL);
     $i->click('[data-automation-id="settings-submit-button"]');
     $i->waitForText('Settings saved');
     $i->reloadPage();
     $i->cantSee($errorMessagePrefix);
   }
-
+/*
```
Comment (lysyjan) [inline mailpoet/tests/acceptance/Settings/AuthorizedEmailAddressesValidationCest.php]: Is there a reason why those test cases are disabled?

**Review by lysyjan — COMMENTED** [review]
Overall it looks good, but I have noticed two details.

**Comment by websupporter** [comment]
Thank you for catching both @lysyjan. Fixed both issues.

**Review by lysyjan — DISMISSED** [review]

**Comment by veljkho** [comment]
Hi @websupporter 
This probably require rebasing due to conflicts?

**Comment by websupporter** [comment]
rebased @veljkho

**Review by veljkho — APPROVED** [review]

**Pull request merged**